### DISCUSSION TITLE
Is Github's help worth waiting for?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#108005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Unfortunately I was a victim of hacking by a bot, but in short, I lost access to my GitHub account completely and that account is in limbo (no one can modify it and it will remain that way forever...).

From what I read, Github support does not provide any help for account losses with 2FA (in my case I did not have that enabled), but I also read about other users experiences regarding Github support and after more than a week I don't know whether to keep waiting for a response or consider that account lost?

The ticket provides all the evidence of the theft of the account and other data, the good thing is that all my codes are safe but, I don't know whether to wait 2 months for a response that I don't even know if it will be helpful or create one new account and reupload all my codes...

I would rather receive a definitive NO at the moment than an uncertain answer in 2 months.

Ticket: #2570741

**Comment by jane-does-coding** [comment]
If you have created a ticket for Github Support they have to answer within a week, unless stated otherwise (based on my knowledge), if it's been over a week and they didn't contact you back, then you should try to create a new ticket, because there are high chances that your ticket got lost or missed. 

However, in your situation I assume they won't be able to help you much anyway. Github Support doesn't have any access to edit your profile, but I think it's worth waiting anyway, to see what they will respond with.

**Comment by ghost** [comment]
Wow, well, maybe it's time to forget the account... thanks
